libs/shared-atomic-design-components/src/lib/atoms/SocialMediaIcons.tsx

```diff
@@ -4,39 +4,56 @@ import './MediaIcons.scss'
 
 const socialMedia = [
   {
-    icon: 'linkedin',
-    link: 'https://www.linkedin.com/school/redi-school-of-digital-integration/',
+    icon: 'fb',
+    link: 'https://www.facebook.com/redischool/',
+    MUC: 'https://www.facebook.com/redimunich/',
+    NRW: 'https://www.facebook.com/redischoolnrw/'
   },
   {
-    icon: 'twitter',
-    link: 'https://twitter.com/redischool?lang=en',
+    icon: 'meetup',
+    link: 'https://www.meetup.com/en-AU/ReDI-school/',
+    MUC: 'https://www.meetup.com/Tech-Talks-Hosted-by-ReDI-School-Munich/',
+    NRW: 'https://www.meetup.com/ReDI-School-NRW/'
   },
   {
     icon: 'instagram',
     link: 'https://www.instagram.com/redischoolberlin/',
+    MUC: 'https://www.instagram.com/redimunich/',
+    NRW: ''
   },
   {
-    icon: 'fb',
-    link: 'https://www.facebook.com/redischool/',
+    icon: 'linkedin',
+    link: 'https://www.linkedin.com/school/redi-school-of-digital-integration/',
   },
   {
-    icon: 'meetup',
-    link: 'https://www.meetup.com/en-AU/ReDI-school/',
-  },
+    icon: 'twitter',
+    link: 'https://twitter.com/redischool?lang=en',
+  }
 ]
 
-const SocialMediaIcons = () => (
-  <ul className="media-icons__list">
-    {socialMedia.map((item: any) => {
-      return (
-        <li key={item.icon}>
-          <a href={item.link} target="_blank" rel="noopener noreferrer">
-            <Icon icon={item.icon} className="media-icons__icon" />
-          </a>
-        </li>
-      )
-    })}
-  </ul>
-)
+interface Props {
+  name?: string
+}
+
+const SocialMediaIcons = ({name}: Props) => {
+  console.log(`name`, name)
+  const icons = name ? socialMedia.slice(0, 3) : socialMedia;
```

Comment:
What's this logic? If anything is given in the `name` prop we only use a certain portion of the `socialMedia` array, otherwise we use the whole thing? If you're trying to do some kind of filtering, then write it appropriately using methods such as `.filter` and more declarative names. I have no idea what the purpose of the `name` prop is.

Reply:
<img width="491" alt="Screenshot 2021-06-14 at 12 00 56" src="https://user-images.githubusercontent.com/23409802/121874966-348cb080-cd08-11eb-9aea-adbc56706b61.png">

I was talking to Miriam when I updated the links that it would make sense to have all of them displayed in the footer since there is a different one for each location but apparently linkedIn (MUC, NRW) and IG (NRW) is missing. So this was a solution for the footer to have the info available.

I just saw that you added a pre-select before being forwarded to your city ... does it maybe make sense to have the links behind the icons according to what location you picked before? On the other hand you maybe want to check out the other locations info as well ... @miriam-aha I guess this decision is mostly up to you? :)

Reply:
Let's leave this up to @miriam-aha. 

Personally though, I think your idea is the best one I've heard. The current design with three facebook icons, three meet-ups, two instagrams, one linkedin, one twitter, and two abbreviated locations (unclear whether they're links or not) is just really confusing.

Reply:
@ericbolikowski I think we have a misunderstanding here: Miriam already agreed on it when we had the call with Paula :)

Reply:
OK, great - I assume I'll see the final visual outcome in your next push then :)